mono/metadata/appdomain.c

```
@@ -334,12 +345,23 @@ void
 mono_context_init (MonoDomain *domain)
 {
 	MonoError error;
+	mono_context_init_checked (domain, &error);
+	mono_error_raise_exception (&error);
```

Same thing. This is an API function. Swallow the error and ban it.